apps/web/src/components/ClickableLogo/ClickableLogo.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import ClickableLogo, { Props } from './ClickableLogo';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/ClickableLogo',
+  component: ClickableLogo,
+  argTypes: {
+    onClick: { action: 'clicked' },
+  },
+};
```

Comment:
There should be additional typing here.
```suggestion
} as ComponentMeta<typeof ClickableLogo>; 
```

Reply:
For more on the `as` type assertion in TypeScript see:

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions